scripts/cram

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# if we have cram already, use that
```

Comment:
Yea, I was torn -- we already install cram for GC, and it's highly unlikely cram with change their format, so do we keep it around to guarantee compat or because we want to test w/o depending on python packages.